src/renderer/api/kube-api.ts

```diff
@@ -35,6 +36,7 @@ export interface IKubeApiOptions<T extends KubeObject> {
 export interface IKubeApiQueryParams {
   watch?: boolean | number;
   resourceVersion?: string;
+  resourceVersionMatch?: "Exact" | "NotOlderThan" // see also: https://kubernetes.io/docs/reference/using-api/api-concepts/#resourceversion-in-metadata
```

Comment:
I think we cannot trust these on older kube versions.

Reply:
This should be backward compatible as far as I know 🤔